Gruntfile.js

```diff
@@ -123,9 +123,6 @@ module.exports = function(grunt) {
       ngCookies: {
         files: { src: 'src/ngCookies/**/*.js' },
       },
-      ngLocale: {
```

Comment:
The function `pluralCat` that comes from google closure does not pass the jshint.